ui/src/components/SaveButton.js

```diff
@@ -152,28 +157,48 @@ class SaveButton extends React.Component {
       if (error) {
         alert(error.response.body.detail);
       } else {
-        let importUrl =
-          "https://app.terra.bio/#import-data?format=entitiesJson";
+        let queryStr = "format=entitiesJson";
```

Comment:
I feel like we shouldn't be manually appending strings together when URLSearchParams exist. I think we should change this to:
`let queryParams = new URLSearchParams("format=entitiesJson")`

And then when adding to it, do (ex):
`queryParams.append("wid", wid)`

And then when calling sendMessage, do:
`queryParams.toString()`

Reply:
IMHO we should use URLSearchParams when it makes the code more readable/better. In this case, I think it actually makes the code a little less readable -- since the code is quite clear on its own.